packages/perseus/src/widgets/dropdown/dropdown.tsx

```diff
@@ -128,11 +128,6 @@ class Dropdown extends React.Component<Props> implements Widget {
                                 this.props.visibleLabel ||
                                 this.context.strings.selectAnAnswer
                             }
-                            // This is currently necessary for SRs to read the labels properly.
-                            // However, WB is working on a change to add the "combobox" role to
-                            // all dropdowns.
-                            // See https://khanacademy.atlassian.net/browse/WB-1671
-                            role="combobox"
```

Comment:
Love it!!

Reply:
Love to remove code and have the tests still pass!!